plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -726,21 +727,21 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
                 setRollback(() -> deleteRecursivelyIfExists(new HdfsContext(session), hdfsEnvironment, deltaLogDirectory));
                 transactionLogWriter.flush();
             }
+            else {
+                if (!isCreateTableWithExistingLocationEnabled(session)) {
+                    throw new TrinoException(
+                            NOT_SUPPORTED,
+                            "Using CREATE TABLE with an existing table is deprecated, instead use the system.register_table() procedure." +
```

Comment:
The table actually not exists, but rather its content.

Consider using a more expressive message.

e.g.: 
_Using CREATE TABLE with an existing table content is deprecated_